doc/source/deployment/verify.rst

```diff
@@ -26,40 +26,30 @@ OpenStack Tempest tests against the core services in the deployment using the `r
 Before running Tempest tests, it will be necessary to manually configure OpenStack network
 resources and provide a few configuration parameters in the ${WORKDIR}/env/extravars file.
 
-Setting Up An External Network And Subnet in OpenStack
+Setting Up Internal/External Network CIDR in OpenStack
 ------------------------------------------------------
 
-To set up an external network and subnet in OpenStack, the following commands can be run from a
-shell on the `Deployer` node.
-
-.. code-block:: console
-
-   export OS_CLOUD=openstack
-   openstack network create --provider-network-type flat --provider-physical-network external \
-     --external public
-   openstack subnet create --network public --subnet-range 192.168.100.0/24 --allocation-pool \
-     start=192.168.100.10,end=192.168.100.200 --gateway 192.168.100.1 --no-dhcp public-subnet
+Tempest will create a private network (10.0.0.0/8) to use as the default network, and it will
+need to know the CIDR block from which to allocate project IPv4 subnets. This value should be
+specified with the following key in the ${WORKDIR}/env/extravars file:
 
-.. note::
+.. code-block:: yaml
 
-   The external public network is expected to be able to reach the internet. The above values
-   will vary based on your network environment.
+   openstack_project_network_cidr: "10.0.4.0/24"
 
-Once the public network and subnet have been created in OpenStack, their names will need to be
-made known to Tempest by adding the following keys in the ${WORKDIR}/env/extravars file:
+By default, same CIDR block is used for creating external network which is needed by tempest
```

Comment:
I thought the code allows the end user (not ECP) to use an existing external network if the user decides to or needs to? should we explain that approach here for end users?

Reply:
Which openstack external network here you are referring to? 

Are you referring to ansible variables _openstack_external_network_name_ and _openstack_external_subnet_name_ ?

I did not mention those variables as then user need to be aware of network/subnets create command in case its created differently (from what's needed for tempest). And may be we end up describing details here as that will come up in when verifying these docs.

I do see your point as it can be useful if compatible existing network can be used. Though not sure what kind of cleanup is done once execution is completed.

Can you clarify above and where it can be useful?

Reply:
I see two scenarios when the user runs tempest. One is that the user has not network created yet (such as in ecp), so convenient/helpful for the playbook to create the network automatically and clean up after the test run is finished. The other case is that the user creates the external network by hand as it is more specific to their specific environment and likely to continue to use it beyond tempest testing. In the latter case, the tempest run will use the already created external network.

Reply:
or if the user reruns the tempest after the cloud used for a while to verify the health/trouble shooting etc., at which time the external network will be pre-existing?

Reply:
Okay. I will add back the creation of network step and providing that information in extravars